bookkeeper-stats-providers/twitter-ostrich-provider/pom.xml

```diff
@@ -32,6 +32,38 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.libdir>${basedir}/lib</project.libdir>
   </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
" it is better to add checkstyle only those projects those are fixed. After every project finished, it can be moved to the root.
In this way you can't left anything unchecked."

+1